Classes/Queue/DoctrineQueue.php

```diff
@@ -191,15 +200,38 @@ protected function reserveMessage(?int $timeout = null): ?Message
         }
         $this->reconnectDatabaseConnection();
 
+        $qb = $this->connection->createQueryBuilder();
```

Comment:
We could reduce some duplication with a
```php
private function createQueryBuilder(): QueryBuilder
{
    return $this->connection->createQueryBuilder()
        ->from($this->tableName)
        ->orderBy('id');
}
```

Reply:
While `orderBy('id')` appears only twice, a "method for `from()`" can be used six times. 👍